doc/Development_Documentation/04_Assets/01_Working_with_PHP_API.md

```diff
@@ -64,7 +64,15 @@ therefore used on all output channels.
 
 ### Examples
 ##### Getting Data
- ```php
+Metadata for an asset can be fetched using `getMetadata` method of Asset class. It takes 4 parameters: 
+1. `$name` can be a nullable string which corresponds to the name of metadata. If null is passed, it tries to get all the metadata. The actual contents may be manipulated by other parameters.
+2. `$language` is a nullable string which corresponds to language of the metadata. If language is passed, it tries to fetch metadata in a specific language.
+3. `$strictMatch` is a boolean value which, if true tries to get only the metadata which exactly matches the requested language.
```

Comment:
Maybe this parameter can be renamed to better reflect what is does, like `$strictLanguageMatch`.

Reply:
I agree, without specifying language, it could be "confused" for some strict matching metadata name feature, like if it would be possible to fuzzy search eg. `copy*`.

Reply:
Renamed the flag to `$strictMatchLanguage`. Thanks @jdreesen for the feedback!